packages/fxa-settings/src/components/PageDeleteAccount/index.tsx

```diff
@@ -115,25 +115,9 @@ export const PageDeleteAccount = (_: RouteComponentProps) => {
             <Localized id="delete-account-confirm-title-2">
               <p className="mb-4">
                 You've connected your Firefox account to Mozilla products that
-                keep you secure and productive on the web:
+                keep you secure and productive on the web.
```

Comment:
You will also need to update text for localizers https://github.com/mozilla/fxa/blob/073e43abeafa5ae3f64e189ccee4d5ec3d2cae80/packages/fxa-settings/src/components/PageDeleteAccount/en-US.ftl#L9.

Reply:
Ack! I'm so sorry, somehow some of my changes must not have made it up. Updating now!

Reply:
Nope I was wrong, I had just failed to do this 😓. Thank you!!